zhishi-project/Zhishi-Frontend
# #137791351 Integrate Bugsnag For Error Reporting

#### What does this PR do?
This PR integrates Bugsnag
#### Description of Task to be completed?
Integration of an error reporting service called bugsnag. This should report production errors and app crashes
#### How should this be manually tested?
This can only run on production.
To test follow the steps below:
- Clone the repo
- Navigate to any react component file
- Manually throw an error like below
```js
throw new Error("This is a manual error");
```
- After save, run this command in your terminal/command-line
`npm run build`
- After the build is done deploy the app.
- Check your bugsnag account
#### Any background context you want to provide?
##### P.S
Before running `npm run build`, add some env variables
```js
BUGSNAG_API='d23kj23jn2nj3nh13923bv423b23jb4j'
NODE_ENV='production'
````
#### What are the relevant pivotal tracker stories?
#137791351
#### Screenshots (if appropriate)
#### Questions:
